lib/gradebook.rb

```diff
@@ -0,0 +1,38 @@
+require 'pry'
+class Gradebook
+  attr_reader :name, :courses
+
+  def initialize(name)
+    @name = name
```

Comment:
Youre storing the name but the spec wants instructor. Make sure that you're making sure that your work matches the spec.